src/main/java/uk/gov/hmcts/reform/migration/ccd/CoreCaseDataService.java

```diff
@@ -21,6 +24,8 @@ public class CoreCaseDataService {
     private AuthTokenGenerator authTokenGenerator;
     @Autowired
     private CoreCaseDataApi coreCaseDataApi;
+    @Autowired
+    private DataMigrationService<Map<String, Object>> dataMigrationService;
 
     public CaseDetails update(String authorisation, String eventId,
```

Comment:
I'd suggest not passing the full case details to this method but just the case id and jurisdiction needed to perform the start event call, to ensure nobody is tempted to make use of the potentially stale case data.

Reply:
Updated to pass caseId and jurisdiction to CoreCaseDataService.update() instead of potentially stale case data.

Reply:
@banderous @danlysiak - changes will be released once you approve